packages/remix-azure-functions/magicExports/remix.ts

```diff
@@ -0,0 +1,7 @@
+// Re-export everything from this package that is available in `remix`.
```

Comment:
This will fix the ESLint error
```suggestion
/* eslint-disable import/no-extraneous-dependencies */

// Re-export everything from this package that is available in `remix`.
```